.secrets.baseline

```diff
@@ -100,33 +109,225 @@
     },
     {
       "path": "detect_secrets.filters.heuristic.is_templated_secret"
-    },
-    {
-      "path": "detect_secrets.filters.regex.should_exclude_secret",
-      "pattern": [
-        "(admin|mp6yc0UD9e|alfresco)"
```

Comment:
maybe using this is even better compared to pre-commit config arg

Reply:
I'd rather we: 
* keep it for convenience
* document theese password (at least admin & alfresco) as dummy ones